scripts/server.sh

```diff
@@ -46,7 +44,7 @@ while [[ ! -e stopserver ]]; do
 
 	# Compile
 	echo "Compiling..."
-	DMoutput="$(./scripts/dm.sh $EXTRA_DM_SH_ARGS -M$MAP $DME.dme | tee /dev/fd/5)" # duplicate output to fd 5 (which is redirected to stdout at the top of this script)
+	DMoutput="$(./scripts/dm.sh $EXTRA_DM_SH_ARGS -M$MAP $DME.dme | tee /dev/fd/2)" # duplicate output to fd 2 for logging purposes
```

Comment:
fd 2 is stderr which is displayed by default, isn't it? then again fd 5 is duplicated to fdout. mostly i'm worried this will end up throwing extra junk into the log file, though maybe that's okay? if it isn't then the only solution is to make it mandatory to run with root permissions.

Reply:
well if it works i guess it's fine